src/test/java/org/jenkinsci/plugins/casc/AgentProtocolsTest.java

```diff
@@ -22,7 +22,7 @@
 
     @Test
     public void configure_agent_protocols() throws Exception {
-        ConfigurationAsCode.configure(getClass().getResourceAsStream("AgentProtocolsTest.yml"));
+        new TestConfiguration("AgentProtocolsTest.yml").configure(getClass());
```

Comment:
Does not help much, it should be rather done with a custom Rule and annotation

Reply:
just fast one-to line replacement, but can do if it would be appreciated in this pr

Reply:
rule added